### DISCUSSION TITLE
Metricbeats install on windows

### DISCUSSION THREAD

**Discussion by afmiller** [Security-Onion-Solutions/securityonion#3991] [Unsupported Versions]
Hello,
Newer to ELK and trying to tinker around with security onion. I am trying to install metricbeats on a windows machine but running into a road block that I don't know how to solve. I am following the guide under Kibana and trying to complete ".\metricbeat.exe setup". I was able to get the windows machine talking to elastic, but when it gets to the kibana part it is failing. It won't load the dashboards. Getting an HTTP GET error. I tried looking under the so-allow but kibana isn't one of the ports I can change with the firewall via so-allow. What am I missing? 

Thanks for the help

**Comment by lan94** [comment]
Hi @afmiller ,

Kibana sits behind a reverse proxy, which means that you cannot simply connect Metricbeat to Kibana. This reverse proxy is used to serve its own authentication.
One solution would be to change the nginx configuration of your nginx container on your Security Onion and make Kibana available directly on a different external port. You could reuse the actual Kibana part of your nginx configuration and remove the authentication part and change the port to your choice.

One should take care that this configuration should be active only during the setup of Metricbeat and should be removed after the transfer of the dashboards.
